core/neat/mixins/_grid-media.scss

```diff
@@ -1,8 +1,15 @@
 @charset "UTF-8";
-/// Creates a media query in which custom grid properties can be defined.
-/// Add the `media` property to your custom grid settings map and add the media
-/// you would like to scope to. If only a number is defined, it is assumed this
-/// is a `min-with` value. Your custom grid can then be passed to the mixin.
+/// Neat supports complex changes to the grid applied by Neat's other mixins
```

Comment:
I’m having a hard time understanding this sentence.

“…supports complex changes…” is too abstract, I think?

Thinking out loud:

- “Neat allows you to _apply_ a grid…”
- You apply a grid via `grid-media`…
- Custom defined grids are scoped…
- You have the option to define custom, unique grids scoped to specific media queries…

It’s definitely a tricky thing to explain and type out!